.changelog/09dc2a7906374b818a70a25a4b238623.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "09dc2a79-0637-4b81-8a70-a25a4b238623",
+    "type": "bugfix",
+    "description": "remove url path cleaning logic during http request serialization and deprecate its setting",
```

Comment:
We don't use the changelog tool in v1, you have to add an entry to `CHANGELOG_PENDING.md`.

It should look something like:

```### SDK Bugs
* `rest`: Remove unnecessary path normalization behavior.
  * This behavior would incorrectly mutate request paths with URI-encoded characters, potentially resulting in misrouted requests.
* `config`: Deprecate `DisableRestProtocolURICleaning` config setting.
  * This setting no longer has any effect. REST-protocol paths will now never be normalized after serialization.

Reply:
(to the `SDK Bugs` section)